code/modules/mob/interactive.dm

```diff
@@ -462,18 +552,11 @@
 
 /mob/living/carbon/human/interactive/proc/target_filter(target)
 	var/list/L = target
-	for(var/atom/A in target)
-		if(istype(A,/area) || istype(A,/turf/space))
+	for(var/atom/A in target) // added a bunch of "junk" that clogs up the general find procs
+		if(istype(A,/area) || istype(A,/turf) || istype(A,/obj/machinery/door) || istype(A,/atom/movable/light) || istype(A,/obj/structure/cable) || istype(A,/obj/machinery/atmospherics))

```

Comment:
maybe make this an actual list and use is_type_in_list() instead of a gigantic istype chain